spring-cloud-stream-binder-kafka/src/main/java/org/springframework/cloud/stream/binder/kafka/config/ExtendedBindingHandlerMappingsProviderConfiguration.java

```diff
@@ -40,10 +40,6 @@ public MappingsProvider kafkaExtendedPropertiesDefaultMappingsProvider() {
 			mappings.put(
 					ConfigurationPropertyName.of("spring.cloud.stream.kafka.bindings"),
 					ConfigurationPropertyName.of("spring.cloud.stream.kafka.default"));
-			mappings.put(
```

Comment:
Again, I moved this to the new default mappings provider. Let me know if I am missing something and this is actually being used in the kafka binder module.